### ISSUE TITLE
Backlight / Screensaver timer

### ISSUE THREAD

**Issue by HaViGit** [Edzelf/ESP32Radio-V2#90]
Hi Ed. Can you tell me where to disable the screensaver or change the delay time? I use the blue TFT 128x160 screen.

**Comment by Edzelf** [comment]
There is no screensaver in the software.

**Comment by HaViGit** [comment]
Maybe I used the wrong word, but I mean the backlight of the display, which switches off after a certain time from the software?

**Comment by Edzelf** [comment]
Yes, I completely forgot this feature.  It is BLTIME at line 158.  Now set to 45 seconds.

**Comment by HaViGit** [comment]
Thanks Ed, I love the project!
